models/action.go

```diff
@@ -136,6 +137,12 @@ func (a Action) GetIssueInfos() []string {
 	return strings.SplitN(a.Content, "|", 2)
 }
 
+func (a Action) GetIssueTitle() string {
+	issueID, _ := strconv.Atoi(strings.SplitN(a.Content, "|", 2)[0])
```

Comment:
You can import `github.com/Unknwon/com` and use `com.StrTo().MustInt64`.

Also,

```go
issueID := com.StrTo(a.GetIssueInfos()[0]).MustInt64()
```